src/opera/template/node.py

```diff
@@ -109,55 +104,44 @@ def get_property(self, params):
             raise DataError("HOST is not yet supported in opera.")
         if host != "SELF":
             raise DataError(
-                "Property host should be set to 'SELF' which is the only "
-                "valid value. This is needed to indicate that the property is "
-                "referenced locally from something in the node itself."
+                "Property host should be set to 'SELF' which is the only valid value. This is needed to indicate that "
+                "the property is referenced locally from something in the node itself."
             )
 
         # TODO(@tadeboro): Add support for nested property values.
         if prop in self.properties:
             return self.properties[prop].eval(self, prop)
 
         # Check if there are capability and requirement with the same name.
-        if prop in [r.name for r in self.requirements] and prop \
-                in [c.name for c in self.capabilities]:
-            raise DataError("There are capability and requirement with the "
-                            "same name: '{}'.".format(prop, ))
+        if prop in [r.name for r in self.requirements] \
+                and prop in [c.name for c in self.capabilities]:
+            raise DataError("There are capability and requirement with the same name: '{}'.".format(prop, ))
 
         # If we have no property, try searching for capability.
-        capabilities = tuple(
-            c for c in self.capabilities if c.name == prop
-        )
+        capabilities = tuple(c for c in self.capabilities if c.name == prop)
         if len(capabilities) > 1:
-            raise DataError("More than one capability is named '{}'.".format(
-                prop,
-            ))
+            raise DataError("More than one capability is named '{}'.".format(prop))
 
         if len(capabilities) == 1 and capabilities[0].properties:
-            return capabilities[0].properties[0].get(rest[0]).data
+            return capabilities[0].properties.get(rest[0]).data
```

Comment:
@anzoman this needed to be fixed because I found a trailing comma (see my other comment) that seemed incorrect.

Please verify that my fix is correct :)

Reply:
I think that we should be okay here.